library/common/jni_interface.cc

```diff
@@ -209,7 +209,7 @@ static void jvm_on_error(envoy_error error, void* context) {
   jobject j_context = static_cast<jobject>(context);
 
   jclass jcls_JvmObserverContext = env->GetObjectClass(j_context);
-  jmethodID jmid_onError = env->GetMethodID(jcls_JvmObserverContext, "onError", "([BI)V");
+  jmethodID jmid_onError = env->GetMethodID(jcls_JvmObserverContext, "onError", "(I[BI)V");
```

Comment:
What does this change?

Reply:
The signature of the method. It went from being a void method that took a byte array and an integer to a void method that takes an integer, a byte array, and an integer.